homeassistant/components/tractive/sensor.py

```diff
@@ -61,7 +63,9 @@ def __init__(self, user_id, trackable, tracker_details, unique_id, description):
     @callback
     def handle_hardware_status_update(self, event):
         """Handle hardware status update."""
-        self._attr_native_value = event[self.entity_description.key]
+        if (_state := event[self.entity_description.key]) is None:
```

Comment:
I am not 100% sure `ATTR_LIVE_TRACKING_REMAINING` and `ATTR_TRACKER_STATE` always will be present.
So far it seems that every hw package contains this